### PR TITLE
Quantum bst code quality

### PR DESCRIPTION
**Pull request by dwierichs**
Here I collect some improvements to be made to quantum.bst, many of which are not of high importance or impact.
This also includes cleaning up and commenting the code for future development help.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,15 @@
 
 # LaTeX document class for Quantum
 
-This is version 5.1 of `quantumarticle`, the document class for typesetting articles in Quantum - the open journal for quantum science.
+This is version 6.0 of `quantumarticle`, the document class for typesetting articles in Quantum - the open journal for quantum science.
```
Comment (cgogolin) [inline README.md:5]: Good catch!

File: quantum-template.tex [hunk 2]
```diff
@@ -291,8 +300,6 @@ \subsection{Subsection}
 \end{equation}
 You are free to change this in case it is more appropriate for your article, but a consistent and unambiguous numbering of sections and equations must be ensured.
 
-If you want your appendices to appear in \texttt{onecolumn} mode but the rest of the document in \texttt{twocolumn} mode, you can insert the command \texttt{\textbackslash{}onecolumn\textbackslash{}newpage} before \texttt{\textbackslash{}appendix}.
-
```
Comment (cgogolin) [inline quantum-template.tex:295]: Why remove that?

File: README.md (hunk 1 above)
Comment (cgogolin) [reply README.md:5]: Wait. I actually don't get why this is shown here. This change is already in both the `develop` and the `nolink_error` branch.

**Comment by cgogolin** [comment]
I am a bit scared. The diff I get when comparing the two branches of this PR locally looks good, but the diff here on github is all messed up.

File: README.md (hunk 1 above)
Comment (dwierichs) [reply README.md:5]: I'm quite confused by the entire diff that GitHub is showing. I checked out the develop branch, edited only `quantum.bst` and changed the base branch of this PR to `develop`, but 15 files show diffs.. Maybe it's easiest if I retry manually.

**Comment by dwierichs** [comment]
Seems to be fine over here #132

**Pull request closed without merging**